redfish-core/include/privileges.hpp

```diff
@@ -224,6 +224,13 @@ inline const Privileges& getUserPrivileges(const std::string& userRole)
         static Privileges readOnly{"Login", "ConfigureSelf"};
         return readOnly;
     }
+    if (userRole == "priv-oemibmserviceagent")
+    {
+        static Privileges admin{"Login", "ConfigureManager", "ConfigureSelf",
```

Comment:
static Privileges ibmserviceagent ?

Reply:
Oops.  The variable name is wrong.  The error has no effect on the code.  I'll change it in the next commit.